README.md

```diff
@@ -46,66 +47,77 @@ it("is a basic test with the page", async ({ page }) => {
 The test runner provides browser primitives as arguments to your test functions. Test functions can use one or more of these arguments.
 
 - `page`: Instance of [Page][page]. Each test gets a new isolated page to run the test.
-- `context`: Instance of [BrowserContext][browser-context]. Each test gets a new isolated context to run the test. The `page` object belongs to this context.
-  - `contextOptions`: Default options passed to context creation. Learn [how to modify them](#modify-context-options).
-- `browser`: Instance of [Browser][browser]. Browsers are shared across tests to optimize resources. Each worker process gets a browser instance.
-  - `browserOptions`: Default options passed to browser/launch creation.
+- `context`: Instance of [BrowserContext][browser-context]. Each test gets a new isolated context to run the test. The `page` object belongs to this context. Learn [how to configure](#modify-options) context creation.
+- `browser`: Instance of [Browser][browser]. Browsers are shared across tests to optimize resources. Each worker process gets a browser instance. Learn [how to configure](#modify-options) browser launch.
```

Comment:
"Each worker process gets a browser instance." jumps on me with complexity. Let's move this to a dedicated section.